components/pre_trial_pricing_modal.tsx/content.tsx

```diff
@@ -0,0 +1,293 @@
+                    plan='Professional'
+                    price='$10'
+                    rate={formatMessage({id: 'pretrial_pricing_modal.rate.userPerMonth', defaultMessage: '/user/month'})}
+                    briefing={{
+                        title: formatMessage({id: 'pretrial_pricing_modal.briefing.professional', defaultMessage: 'All the features of Starter, plus'}),
+                        items: [
+                            formatMessage({id: 'pretrial_pricing_modal.briefing.storage', defaultMessage: '{storage}GB file storage'}, {storage: '100'}),
+                            'OneLogin / ADFS SAML 2.0',
+                            'OpenID connect',
+                            'Office365 suite integration',
+                            formatMessage({id: 'pretrial_pricing_modal.briefing.professional.readOnlyAnnoucementChannels', defaultMessage: 'Read-only announcement channels'})],
+                    }}
+                    buttonDetails={{
+                        action: () => {},
+                        text: formatMessage({id: 'pretrial_pricing_modal.btn.upgrade', defaultMessage: 'Upgrade'}),
+                        customClass: ButtonCustomiserClasses.active,
+                    }}
+                    planLabel={
+                        <PlanLabel
+                            text={formatMessage({id: 'pretrial_pricing_modal.planLabel.mostPopular', defaultMessage: 'MOST POPULAR'})}
+                            bgColor='#1E325C'
+                            color='#FFFFFF'
+                            firstSvg={<StarMarkSvg/>}
+                            secondSvg={<StarMarkSvg/>}
+                        />}
+                />
+                <Card
+                    topColor='#1C58D9'
+                    plan='Enterprise'
+                    price={formatMessage({id: 'pretrial_pricing_modal.price.contactSales', defaultMessage: 'Contact sales'})}
+                    briefing={{
+                        title: formatMessage({id: 'pretrial_pricing_modal.briefing.enterprise', defaultMessage: 'All the features of Professional, plus'}),
+                        items: [
+                            'LDAP group sync',
+                            formatMessage({id: 'pretrial_pricing_modal.briefing.professional.highAvailability', defaultMessage: 'High Availability'}),
+                            formatMessage({id: 'pretrial_pricing_modal.briefing.enterprise.advancedCompliance', defaultMessage: 'Advanced compliance'}),
+                            formatMessage({id: 'pretrial_pricing_modal.briefing.professional.advancedRolesAndPermissions', defaultMessage: 'Advanced roles and permissions'})],
+                    }}
+                    buttonDetails={{
+                        action: () => {},
+                        text: formatMessage({id: 'pretrial_pricing_modal.btn.tryDays', defaultMessage: 'Try free for {days} days'}, {days: '30'}),
+                        customClass: ButtonCustomiserClasses.special,
```

Comment:
Once we get to it in a future effort, I think this will need to be disabled if the user is on professional OR enterprise plans, because I don't think we support downgrade yet.

Reply:
For now I am going to keep it disabled always until we support downgrading